src/pages/ErrorPage/index.js

```diff
@@ -0,0 +1,15 @@
+import React from "react"
+import "./Error.scss"
+
+import Layout from "../../components/Layout"
+
+const index = () => (
+  <Layout>
+    <div className='container'>
```

Comment:
**<div className='container'>** nu are sens aici, poti adauga un props **<Layout className="container">** si sal setezi in interior, unde ai **<section className={classes("layout", {[classname]: !!className}}>**

Nu tre sa wrapuesti ceia ce ai in layout cu alte div-uri

Reply:
Ah da, instaleaza libraria 'classNames' --> **import classes from 'classnames'**

vom folosi asta si mai tarziu in alte locuri